.mkchlog.yml

```diff
@@ -0,0 +1,23 @@
+# mkchlog was introduced after this commit
+skip-commits-up-to: 1eb6e0c7d85fbc5e3f265de069b91be2cb722359
```

Comment:
So we update this commit hash right before merging to the tip of master? If we agree to merge this first after release we could use the tagged v0.31.0 commit.

Reply:
Yes. I guess tag could work too, never tried. (But obviously CI will fail until it's tagged.)